runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -267,7 +266,8 @@ bool handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe)
    bool done = false;
```

Comment:
I would move the line `serverMsgTypeCount[response] += 1;` up so that we can count all messages

Reply:
Not sure I'm following. Move `serverMsgTypeCount[response] += 1;` after `   response = client->read();`?

Reply:
yes. Otherwise the messages that are sent when the client detects an interruption are not counted